ideascube/serveradmin/tests/test_views.py

```diff
@@ -642,3 +642,29 @@ def test_staff_forgets_non_existing_connection(mocker, staffapp):
     assert u'No known Wi-Fi networks' not in res.unicode_body
     assert u'my home network' in res.unicode_body
     assert u'random open network' in res.unicode_body
+
+
+@pytest.mark.usefixtures('installed_packages_catalog')
+def test_staff_should_see_packages_with_cards(staffapp, systemuser):
+    from ideascube.configuration import set_config
+    set_config('ideascube', 'displayed-package-ids', ['test.package1'], systemuser)
+    res = staffapp.get(reverse("server:cards"), status=200)
+    assert 'Test package1' in res.unicode_body
+    assert 'Test package2' in res.unicode_body
+    assert 'Test package3' not in res.unicode_body
+
+    form = res.forms['cards']
+    inputs = form.fields['visible']
+    for _input in inputs:
+        should_be_checked = (_input._value == 'test.package1')
```

Comment:
> Well, no, it could be used also in the CSS, the Javascript, ... That's really very standard HTML.

In our case, it will be use only by tests.

> It would be like trying to identify an element of a dictionary by its value, rather than by its key.

I do not try to identify a field from its value. I loop over all the input field named 'visible' and a do different test depending of their 'value'

> I really don't see why you wouldn't be testing the right thing here.

HTML form use the `value` attribute. This is how the POST list is constructed.
`id` is not used at all by django.
If I test `id` and not `value`, I don't test an attribute used and I test an attribute not used.